### PR TITLE
fix: ADVISOR-1943 - Update empty states

### PR DESCRIPTION
**Pull request by bastilian**
This updates the empty states mentioned in the Jira.

The Jira also describes the conditions that the empty states should occur in and where.

### CODE REVIEW

File: src/PresentationalComponents/SystemsTable/Components/NoSystemsTable.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  TextVariants,
+  Title,
+  Bullseye,
+} from '@patternfly/react-core';
+
+const NoSystemsTable = () => (
+  <Bullseye>
+    <EmptyState variant={EmptyStateVariant.full}>
+      <Title headingLevel="h2" size="lg" style={{ fontWeight: 'bold' }}>
+        No matching systems found
```
Comment (mkholjuraev) [inline src/PresentationalComponents/SystemsTable/Components/NoSystemsTable.js:17]: Should we start adding these texts to internationalizatio (**intl.formatMessage()**)

File: src/Messages.js [hunk 2]
```diff
@@ -142,7 +142,13 @@ export default defineMessages({
   ruleIsDisabledBody: {
     id: 'ruleIsDisabledBody',
     description: 'Explaining that the recommendationis disabled',
-    defaultMessage: 'This recommendation has been disabled and has no results.',
+    defaultMessage: 'This recommendation was disabled and has no results.',
+  },
+  ruleIsDisabledBodyWithJustification: {
+    id: 'ruleIsDisabledBodyWithJustification',
+    description: 'Explaining that the recommendationis disabled',
```
Comment (mkholjuraev) [inline src/Messages.js:149]: Minor: there is a spelling error on line: 144

**Review by mkholjuraev — CHANGES_REQUESTED** [review]
There is one scenario which I think not resolved **Advisor Topics recommendations ->  details page:**

it should show **To find more recommendations,**  but PR shows **To view more recommendations, **

I also did not understand how to test the scenario **2. The suggested rewrite when the recommendation is disabled and the user wrote a reason to a justification note**

File: src/PresentationalComponents/SystemsTable/Components/NoSystemsTable.js (hunk 1 above)
Comment (bastilian) [reply src/PresentationalComponents/SystemsTable/Components/NoSystemsTable.js:17]: Yeah, let's start adding them to translations. I'll pull out the strings where I find them for this PR.

File: src/Messages.js (hunk 2 above)
Comment (bastilian) [reply src/Messages.js:149]: Nice catch! Fixed that!

**Comment by bastilian** [comment]
> There is one scenario which I think not resolved Advisor Topics recommendations -> details page:

This should now be corrected. 

>  The suggested rewrite when the recommendation is disabled and the user wrote a reason to a justification note

This should be on the details page of a disabled recommendation. You'll probably need to enter a justification for a recommendation and disable it or use an already disabled one.

**Review by mkholjuraev — APPROVED** [review]

**Pull request merged**